### ISSUE TITLE
Fix Accessibility Warnings

### ISSUE THREAD

**Issue by richarddubay** [NewSpring/Holtzman#1699]
We just added a library that will throw warnings through eslint when our code doesn't meet accessibility standards. This will help us to have a more friendly and usable site/app to those with special needs. These warnings need to be fixed. 

Steps to reproduce:
1. Clone Holtzman and make sure you're on the master branch.
2. Run `npm run lint` and see all the errors.
3. Fix them.
4. Celebrate like the king or queen you are. 🎉

**Comment by briankalwat** [comment]
Putting these here for reference:
- https://www.smashingmagazine.com/2014/09/making-modal-windows-better-for-everyone/
- https://www.marcozehe.de/2015/02/05/advanced-aria-tip-2-accessible-modal-dialogs/

**Comment by meredithrouse** [comment]
SYS-4082
